VoxelSniperForge/src/main/java/com/github/kevindagame/voxelsniperforge/material/BlockMaterial.java

```diff
@@ -29,8 +29,8 @@ public Block getBlock() {
         return block;
     }
 
-    private Material getMaterial() {
-        return block.defaultBlockState().getMaterial();
+    private BlockState getMaterial() {
```

Comment:
Should we also rename this method?

Reply:
Good point. Hadn't thought about it, but it only has references within forge